client/src/types.d.ts

```diff
@@ -57,11 +56,10 @@ export interface DocumentSettings {
 }
 
 export interface DenoExtensionContext {
-  client: LanguageClient;
+  client: LanguageClient | undefined;
```

Comment:
When the language server doesn't start then this will be undefined.

Reply:
Ok, now I see it undefined possibly here...

Reply:
Sorry for the confusion, I renamed the title to something more clear (and correct).